src/arch/esp32.h

```diff
@@ -63,18 +65,6 @@
 
 #elif defined(CIRCUITPY) // COMPILING FOR CIRCUITPYTHON --------------------
 
-// ESP32 requires a custom PEW declaration (issues one set of RGB color bits
-// followed by clock pulse). Turns out the bit set/clear registers are not
-// actually atomic. If two writes are made in quick succession, the second
-// has no effect. One option is NOPs, other is to write a 0 (no effect) to
-// the opposing register (set vs clear) to synchronize the next write.
-#define PEW                                                                    \
-  *set = (*data++) << shift; /* Set RGB data high */                           \
-  *clear_full = 0;           /* ESP32 MUST sync before 2nd 'set' */            \
-  *set = clock;              /* Set clock high */                              \
-  *clear_full = rgbclock;    /* Clear RGB data + clock */                      \
-  ///< Bitbang one set of RGB data bits to matrix
-
```

Comment:
Also why this was removed?

Reply:
PEW is used in `src/core.c` and mentioned in comments elsewhere.

Reply:
Fixed now.